pkg/kubernetes/diff.go

```diff
@@ -209,3 +217,13 @@ func (m multiDiff) diff() (*string, error) {
 	}
 	return &diff, nil
 }
+
+func findNamespaces(state manifest.List) map[string]bool {
+	namespaces := map[string]bool{}
+	for _, m := range state {
+		if m.Metadata().HasNamespace() {
```

Comment:
`Namespace()` already considers `HasNamespace`. Would it be cleaner to write it like this?
```
if namespace := m.Metadata().Namespace(); namespace != "" {
  namespaces[namespace] = true
}
```

Reply:
yep, reasonable, done.